notebooks/cosine_io.ipynb

```diff
@@ -0,0 +1,1789 @@
+{
```

Comment:
<p>"...to receive a <strong>pyiron</strong> job <strong>object</strong> back from coscine..."</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pyiron/pyiron_contrib/pull/632/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='53'/>